src/compiler/core.ts

```diff
@@ -703,7 +682,7 @@ export function concatenate<T>(array1: readonly T[], array2: readonly T[]): read
 export function concatenate<T>(array1: T[], array2: T[] | undefined): T[]; // eslint-disable-line @typescript-eslint/unified-signatures
 /** @internal */
 export function concatenate<T>(array1: T[] | undefined, array2: T[]): T[]; // eslint-disable-line @typescript-eslint/unified-signatures
-/** @internal */
+/** @internal *

```

Comment:
The heck is happening here

Reply:
I dunno how that happened, but it is pretty funny that this is never used.